.github/workflows/ci.yml

```diff
@@ -1,7 +1,9 @@
 ---
 name: CI
 
-on: push
+on:
+  push:
```

Comment:
@JoseLion maybe I'm wrong, but since there is no branch filter for the `push` event I think this workflow is going to be triggered twice in the context of the `release` branch:

1st trigger - `push` event on the `release` branch
2nd trigger - `uses: ./.github/workflows/ci.yml` in the `release` workflow

Reply:
Thet's right! Nice catch @sestrella! I'll add a filter to ignore `release` in the `push` event 🙂